tmva/sofie/inc/TMVA/ROperator_BasicBinary.hxx

```diff
@@ -323,7 +330,7 @@ public:
             if (fShapeA[i] == 1)
                continue;
             compute_idx_A +=
-               " idx_" + fNY + std::to_string(i + (fShapeY.size() - fShapeA.size())) + " * " + stridesA[i] + " +";
+               " idx_" + std::to_string(i + (fShapeY.size() - fShapeA.size())) + " * " + stridesA[i] + " +";
```

Comment:
I think we have some of the changes from the other PR on Basic Binary operators, these may cause a merge conflict later.

Reply:
I have removed this commit which is part of #19729